### ISSUE TITLE
Client-side changes to support room versioning

### ISSUE THREAD

**Issue by lampholder** [vector-im/riot-meta#212]
We've not decided how the server is going to implement room versioning yet, but a few of the conceivable options include the clients' having some work to do.

This issue is here to track that possibility.

**Comment by erdnaxeli** [comment]
The google doc is not world readable, is it normal? And I can't open the figma doc neither.

**Comment by ara4n** [comment]
no, oversight. have turned on link sharing for the gdoc but don’t have access on figma

**Comment by lampholder** [comment]
@erdnaxeli the figma doc looks like it should be globally accessible now.

**Comment by Siilwyn** [comment]
Is it possible to do this at the moment by other means?
